app/assets/javascripts/components/forms/checkBox.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { PureComponent } from 'react';
+import { Field } from 'redux-form';
+import { Checkbox } from 'react-bootstrap';
```

Comment:
Do we need this import? Doesn't seem like we're using it as we are defining `CheckBox` below. Also, having something called `Checkbox` here and `CheckBox` below is confusing.